src/main/resources/com/atlassian/jira/cloud/jenkins/Messages.properties

```diff
@@ -1,5 +1,7 @@
 JiraSendBuildInfoStep.DescriptorImpl.DisplayName=Atlassian Jira Software Cloud Jenkins Integration (Build)
 JiraSendDeploymentInfoStep.DescriptorImpl.DisplayName=Atlassian Jira Software Cloud Jenkins Integration (Deployment)
+JiraCheckGateStatusStep.DescriptorImpl.DisplayName=Atlassian Jira Software Cloud Jenkins Integration (Deployment Gate)
```

Comment:
Where is this message shown? This definitely shouldn't say `Jira Software` since deployment gating is a `Jira Service Desk` feature. But depending on where it's shown it might be confusing for the step that submits the deployment (which I assume is the message above) to be `Jira Software` when it's also used for the Jira Service Desk / Deployment gating feature/functionality.

Reply:
it displays only in blue ocean plugin. This how it shows after changes 
<img width="2028" alt="image" src="https://user-images.githubusercontent.com/13866979/96585847-cf073200-12e8-11eb-9c9f-c11be6b97718.png">
I think this's a little mess after changes. What do u think @aolrich323